src/main/java/appeng/me/storage/BasicCellInventory.java

```diff
@@ -201,9 +224,9 @@ public IAEItemStack extractItems( final IAEItemStack request, final Actionable m
 
 		final long size = Math.min( Integer.MAX_VALUE, request.getStackSize() );
 
-		IAEItemStack Results = null;
+		T Results = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00117)